src/lib/css-parse.html

```diff
@@ -89,6 +90,12 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces `\33 ` into
+    // expanded form that doesn't require space `\000033`
+    _expandUnicodeEscapes : function (s) {
+      return s.replace(/\\([0-9]+)\s/g, '\\0000$1');

```

Comment:
@arthurevans, I wanted to avoid using functions initially.
Now it should be fine.